File: javascript/challenge13.js
const lightsOn = function(lights) {
  lights.forEach(light =>{light.on = true});
  return lights;
}

const lightsOff = function(lights) {
  lights.forEach(light =>{light.on = false});
  return lights;
}

const toggleLights = function(lights, lightsAreOn) {
  if(lightsAreOn)
    {
    return lightsOff(lights);
    }
  return lightsOn(lights);
}

File: ruby/challenge13.rb
def lightsOn(lights):
  for light in lights:
    light.on = True
  end
  return lights
end

def lightsOff(lights):
    for light in lights:
      light.on = False
    end
    return lights;

def toggleLights(lights, lightsAreOn):
  for light in lights:
    light.on = ~light.on
  end
  return lights
end
